aware-core/src/main/java/com/aware/Aware.java

```diff
@@ -425,7 +425,7 @@ protected Boolean doInBackground(Void... params) {
                     }
 
                     //Ignored by standalone apps. They handle their own sensors, so server settings do not apply.
-                    if (!getResources().getBoolean(R.bool.standalone)) {
+                    if (getPackageName().equals("com.aware.phone") || !getResources().getBoolean(R.bool.standalone)) {
```

Comment:
The client is not a standalone (resource is set as false) (it saves the data to a shared folder on the sdcard). Standalone apps are apps where there is no client at all on the phone, data is stored on it's own /data/package folder.